_includes/head.html

```diff
@@ -0,0 +1,17 @@
+  <head>
```

Comment:
maybe it is just github, but the indent seems to be off

Reply:
Not GitHub, you are correct. Fixing!